examples/SEA076_20230906T0852_nrt.cdl

```diff
@@ -0,0 +1,389 @@
+		LATITUDE:valid_max = 90LL ;
+		LATITUDE:valid_min = -90LL ;
+	double LONGITUDE(N_MEASUREMENTS) ;
+		LONGITUDE:_FillValue = NaN ;
+		LONGITUDE:coordinate_reference_frame = "urn:ogc:crs:EPSG::4326" ;
+		LONGITUDE:long_name = "longitude" ;
+		LONGITUDE:observation_type = "measured" ;
+		LONGITUDE:platform = "platform" ;
+		LONGITUDE:reference = "WGS84" ;
+		LONGITUDE:standard_name = "longitude" ;
+		LONGITUDE:units = "degrees_east" ;
+		LONGITUDE:valid_max = 180LL ;
+		LONGITUDE:valid_min = -180LL ;
+	double TIME(N_MEASUREMENTS) ;
+		TIME:_FillValue = NaN ;
+		TIME:long_name = "Time" ;
+		TIME:observation_type = "measured" ;
+		TIME:standard_name = "time" ;
+		TIME:URI = "https://vocab.nerc.ac.uk/collection/P02/current/AYMD/" ;
+		TIME:units = "seconds since 1970-01-01T00:00:00+00:00" ;
+		TIME:calendar = "proleptic_gregorian" ;
+	double DEPTH(N_MEASUREMENTS) ;
+		DEPTH:_FillValue = NaN ;
+		DEPTH:source = "pressure" ;
+		DEPTH:long_name = "glider depth" ;
+		DEPTH:standard_name = "depth" ;
+		DEPTH:units = "m" ;
+		DEPTH:comment = "from science pressure and interpolated" ;
+		DEPTH:instrument = "instrument_ctd" ;
+		DEPTH:observation_type = "calculated" ;
+		DEPTH:accuracy = 1LL ;
+		DEPTH:precision = 2LL ;
+		DEPTH:resolution = 0.02 ;
+		DEPTH:platform = "platform" ;
+		DEPTH:valid_min = 0LL ;
+		DEPTH:valid_max = 2000LL ;
+		DEPTH:reference_datum = "surface" ;
+	double LATITUDE_GPS(N_MEASUREMENTS) ;
+		LATITUDE_GPS:_FillValue = NaN ;
+		LATITUDE_GPS:coordinate_reference_frame = "urn:ogc:crs:EPSG::4326" ;
+		LATITUDE_GPS:long_name = "LATITUDE of each GPS location" ;
+		LATITUDE_GPS:observation_type = "measured" ;
+		LATITUDE_GPS:platform = "platform" ;
+		LATITUDE_GPS:reference = "WGS84" ;
+		LATITUDE_GPS:standard_name = "latitude" ;
+		LATITUDE_GPS:units = "degrees_north" ;
+		LATITUDE_GPS:valid_max = 90LL ;
+		LATITUDE_GPS:valid_min = -90LL ;
+		LATITUDE_GPS:coordinates = "DEPTH LATITUDE LONGITUDE TIME" ;
+	double LONGITUDE_GPS(N_MEASUREMENTS) ;
+		LONGITUDE_GPS:_FillValue = NaN ;
+		LONGITUDE_GPS:coordinate_reference_frame = "urn:ogc:crs:EPSG::4326" ;
+		LONGITUDE_GPS:long_name = "LONGITUDE of each GPS location" ;
+		LONGITUDE_GPS:observation_type = "measured" ;
+		LONGITUDE_GPS:platform = "platform" ;
+		LONGITUDE_GPS:reference = "WGS84" ;
+		LONGITUDE_GPS:standard_name = "longitude" ;
+		LONGITUDE_GPS:units = "degrees_east" ;
+		LONGITUDE_GPS:valid_max = 180LL ;
+		LONGITUDE_GPS:valid_min = -180LL ;
+		LONGITUDE_GPS:coordinates = "DEPTH LATITUDE LONGITUDE TIME" ;
+	double TIME_GPS(N_MEASUREMENTS) ;
+		TIME_GPS:_FillValue = NaN ;
+		TIME_GPS:long_name = "TIME of each GPS location" ;
+		TIME_GPS:observation_type = "measured" ;
+		TIME_GPS:standard_name = "time" ;
+		TIME_GPS:URI = "https://vocab.nerc.ac.uk/collection/P02/current/AYMD/" ;
+		TIME_GPS:coordinates = "DEPTH LATITUDE LONGITUDE TIME" ;
+		TIME_GPS:units = "seconds since 1970-01-01T00:00:00+00:00" ;
+		TIME_GPS:calendar = "proleptic_gregorian" ;
+	char TRAJECTORY(string24) ;
```

Comment:
The manual defines trajectory as a `string` type, and by doing so we can avoid these `stringxx` dimensions.

Reply:
That's what I did at first, but I found that it failed the CF1.8 checker https://cfchecker.ncas.ac.uk/

If I'm reading the CF docs correctly, they require strings of set length http://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#_data_types

I'm happy to be corrected on this, but we need to make sure we're compliant with CF. Can we have plain string type with CF1.10?

Reply:
After discussion this morning I have changed these to `string` type. We believe this is compliant with CF1.10